gridcapa-core-cc-app/src/main/java/com/farao_community/farao/gridcapa_core_cc/app/CoreCCListener.java

```diff
@@ -35,85 +33,44 @@
 public class CoreCCListener implements MessageListener {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CoreCCListener.class);
-    private static final String APPLICATION_ID = "core-cc-runner";
-    private static final String CONTENT_ENCODING = "UTF-8";
-    private static final String CONTENT_TYPE = "application/vnd.api+json";
-    private static final int PRIORITY = 1;
     private static final String TASK_STATUS_UPDATE = "task-status-update";
 
     private final JsonApiConverter jsonApiConverter;
-    private final AmqpTemplate amqpTemplate;
     private final CoreCCHandler coreCCHandler;
-    private final AmqpMessagesConfiguration amqpMessagesConfiguration;
     private final StreamBridge streamBridge;
 
-    public CoreCCListener(CoreCCHandler coreCCHandler, AmqpTemplate amqpTemplate, AmqpMessagesConfiguration amqpMessagesConfiguration, StreamBridge streamBridge) {
+    public CoreCCListener(CoreCCHandler coreCCHandler, StreamBridge streamBridge) {
         this.streamBridge = streamBridge;
         this.jsonApiConverter = new JsonApiConverter();
         this.coreCCHandler = coreCCHandler;
-        this.amqpTemplate = amqpTemplate;
-        this.amqpMessagesConfiguration = amqpMessagesConfiguration;
     }
 
     @Override
     public void onMessage(Message message) {
-        String correlationId = message.getMessageProperties().getCorrelationId();
-        // propagate in logs MDC the task id as an extra field to be able to match microservices logs with calculation tasks.
-        // This should be done only once, as soon as the information to add in mdc is available.
         try {
+            // propagate in logs MDC the task id as an extra field to be able to match microservices logs with calculation tasks.
```

Comment:
Move this comment one line below so that it is just next to the MDC instruction (line 53).